backend/src/Database.hs

```diff
@@ -49,11 +58,9 @@ initializeDatabase = do
           \strongholds TEXT NOT NULL,\
           \interestRating INTEGER NOT NULL,\
           \comments TEXT,\
-          \winnerRatingAfter INTEGER NOT NULL,\
-          \loserRatingAfter INTEGER NOT NULL,\
```

Comment:
> I ended up walking away from storing ratings in the report table directly - kind of a headache

This makes sense, now that you mention it, these do seem redundant when you can just find them in the Ratings table, plus you added an index so there's no real drawback. I can't remember what the original rationale was. But anyway was the headache because `insertRatingChange` had to happen after `insertGameReport`?

Reply:
Originally we were talking about not having a ratings table because it's 1:1 with game reports. 

But I found that storing ratings on the game report got awkward to query, because a report row only stores the Side played by the winner. So if you want, say, "Chelsea's latest Shadow rating", you need to query for "the last game report with Side Shadow that Chelsea won and then get winnerRatingAfter, OR the last game report with Side Free that Chelsea lost and then get loserRstingAfter" and this was starting to make me dizzy. 

So I think this keeps things a lot more clear, at least in my head.

Reply:
Now we store two rating rows per report row. One per player, each annotated with Side

Reply:
Gotcha, it was just a leftover from before the Ratings table decision, now I'm caught up! Dizzy from following the old way as well haha